buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -36,6 +36,44 @@
         "cwd": "",
         "failOnStandardError": "false"
       }
+    },
+    {
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Initialize tools",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "scriptPath": "pkg/init-tools.sh",
+        "args": "",
+        "disableAutoCwd": "false",
+        "cwd": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Clean up Docker images and containers.",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "scriptPath": "pkg/Tools/scripts/docker/cleanup-docker.sh",
+        "args": "",
+        "disableAutoCwd": "false",
+        "cwd": "",
+        "failOnStandardError": "false"
```

Comment:
I tried running this def in VSTS and it failed. This step tried to run this script with `bash`, but `cleanup-docker.sh` is really a `perl` script. (The step doesn't execute `./pkg/Tools/...`, but rather `bash pkg/Tools/...`, which ignores the `#!` at the top of the file.)

Changing this to a "command line" step that runs `perl` with arg `pkg/Tools/scripts/docker/cleanup-docker.sh` works.